server.go

```diff
@@ -4215,9 +4215,14 @@ func (s *server) addPeer(p *peer.Brontide) {
 		return
 	}
 
+	pubSer := p.IdentityKey().SerializeCompressed()
+	pubStr := string(pubSer)
+
 	// Ignore new peers if we're shutting down.
 	if s.Stopped() {
+		srvrLog.Infof("Server stopped, skipped adding peer=%s", pubStr)
```

Comment:
This `pubStr` is a raw bytes to string conversion and will NOT look human-readable. We can't change it to a hex encoded string because it is used as a map access below (which is also done in different places, so changing one access only will lead to very weird failures, we've had this in another PR). So I think we need the backward compatible variable and a human-readable one. And a comment describing the problem! (I thought we added one in the last revert of this).

Reply:
Done - def gonna change it in a future PR.